index.html

```diff
@@ -0,0 +1,183 @@
+               </div>
+               <div class="grid-box">
+                 <img src="Capstone/Icons/program_icon_02.png" alt="lecture" class="image-box">
+                 <h3 class="heading-box">CC Exhibition</h3>
+                 <p class="box-para">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+               </div>
+               <div class="grid-box">
+                 <img src="Capstone/Icons/program_icon_03.png" alt="lecture" class="image-box">
+                 <h3 class="heading-box">Forum</h3>
+                 <p class="box-para">Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+               </div>
+               <div class="grid-box">
+                 <img src="Capstone/Icons/program_icon_04.png" alt="lecture" class="image-box">
+                 <h3 class="heading-box">Workshop</h3>
+                 <p class="box-para">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+               </div>
+               <div class="grid-box">
+                 <img src="Capstone/Icons/program_icon_05.png" alt="lecture" class="image-box">
+                 <h3 class="heading-box">CC Ignite</h3>
+                 <p class="box-para">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+               </div>
+           </div>
+               <div class="join-container">
+                 <p id="join-btn">Join CC Globel Summit 2015</p>
+                 <p id="see-program">See the whole Program</p>
+                </div>
+
+   </section>
+   <!-- speakers section -->
+   <section class="speaker">
+        <div class="speaker-container">
+            <h2>Featured Speakers</h2>
+            <div class="speaker-one">
+              <div class="speaker-close">
+               <div class="box-image">
+                 <img src="Capstone/speakers/speaker1.png" alt="speaker1" class="speaker-image">
+                </div>
+                <div class="text-speaker">
+                  <h4>Yochai bankler</h4>
+                   <p class="speaker-para">Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita.</p>
+                   <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto ab porro quibusdam.</p>
+                </div>
+               </div>
+               <div class="speaker-close">
+                <div class="box-image">
+                 <img src="Capstone/speakers/speaker1.png" alt="speaker1" class="speaker-image">
+                </div>
+               <div class="text-speaker">
+                  <h4>yochai bankler</h4>
+                  <p class="speaker-para">Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita.</p>
+                  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto ab porro quibusdam.</p>
+               </div>
+             </div>
+             <div class="more-container">
+               <a href="#" class="more-btn">More <img src="capstone/Header/arrow_down.png" alt=""></a>
```

Comment:
- [ ] Kindly remove this section. You have to add it via javascript, same as how you have added it to the desktop version.